stubs/jsonpath_ng/jsonpath.pyi

```diff
@@ -0,0 +1,11 @@
+from pydantic import JsonValue
```

Comment:
Feels a bit weird to me to pull in pydantic for this, but for vendored stubs I guess it suffices.